### ISSUE TITLE
WebVR integration

### ISSUE THREAD

**Issue by mkeblx** [mkeblx/x3dom#1]
Integrate WebVR rendering into x3dom. See previously made/adapted examples that enabled WebVR 1.0 support (https://github.com/mkeblx/x3d-webvr-demo). 

But this is cumbersome: need to create rendert argets for stereo rendering, include WebVR JS code for cameras, run loop, etc. Can more easily adapt into by adding an attribute to root `x3d` tag or most appropriate ~top-level tag i.e. like `viewpoint`.

**Comment by mkeblx** [comment]
PR for inclusion soonish

**Comment by mwitchwilliams** [comment]
I'm developing the X3D parsing for GearVR.
Just reviewed the <Viewpoint> node in the X3D spec. and didn't see anything for stereo rendering.
Though reviewed some work by Leonard Daly (http://realism.com/x3d/stereo) and he has a mix of CSS and <Script> to create the stereoscopic view.

**Comment by mkeblx** [comment]
Yes I was just going to put a `stereo` attribute somewhere and detect it as a placeholder, which can move easily. It is more a property of the user agent/loader/library and not the scene, so maybe a better way to handle more clearly.
